src/main/java/com/rcv/FileUtils.java

```diff
@@ -45,6 +46,8 @@ static void createOutputDirectory(String dir) throws UnableToCreateDirectoryExce
       // dirFile is the File object for dir
       File dirFile = new File(dir);
       if (!dirFile.exists() && !dirFile.mkdirs()) {
+        Logger.log(Level.SEVERE, "Failed to create output directory: %s", dir);
+        Logger.log(Level.SEVERE, "Check the directory name and permissions.");
         throw new UnableToCreateDirectoryException("Unable to create output directory: " + dir);
```

Comment:
The logging you added here isn't redundant with this exception we're throwing? It looks like we catch this exception and then log the same things in TabulatorSession.

Reply:
Fixed.